rest-api/src/test/java/io/quarkus/qe/RepositoryResourceTest.java

```diff
@@ -32,8 +35,10 @@
 public class RepositoryResourceTest {
 
     private static final String PATH = "/repository";
-    private static final String REPO_URL = "http://github.com/user/repo.git";
+    private final static String GIT_EXTENSION = ".git";
+    private static final String REPO_URL = "http://github.com/user/repo" + GIT_EXTENSION;
     private static final long AN_ENTITY_ID = 100;
+    private final static int AMOUNT_RANDOM_REPOS = 3;
```

Comment:
I think that having the RANDOM in the field name is confusing as 3 is a fixed value. I would name it as "REPOS_COUNT" or alike.